packages/components/package.json

```diff
@@ -71,15 +75,20 @@
     "@kaizen/select": "^6.13.6",
     "@kaizen/split-button": "^1.3.33",
     "@kaizen/tabs": "^1.5.30",
-    "@kaizen/tailwind": "^0.5.1",
     "@kaizen/typography": "^2.3.6",
     "classnames": "^2.3.2"
   },
   "devDependencies": {
-    "@deanc/esbuild-plugin-postcss": "^1.0.2",
+    "@kaizen/tailwind": "^0.5.1",
     "autoprefixer": "^10.4.13",
+    "concat-cli": "^4.0.0",
+    "cssnano": "^5.1.15",
```

Comment:
What's this for? I only see it as a key in the postcss config (or is having it as a key enough to reference the package?)

Reply:
PostCSS is a strange and wonderful config file where merely referencing the key enables the plugin 😅